src/phases/execution/MixedMultiDeletes.yml

```diff
@@ -0,0 +1,304 @@
+- RunCommand
+- Loader
+- LoggingActor
+- CrudActor
+- insert
+- delete
+- batch
+- deleteMany
+- latency
+
+GlobalDefaults:
+
+  #### User Defined Parameters ####
+
+  # Whether to use the BATCHED_DELETE query exec stage over DELETE.
+  UseBatchedDeleteStage: &UseBatchedDeleteStage {^Parameter: {Name: "UseBatchedDeleteStage", Default: 0}}
+
+  # Configuration file to use: with no secondary indexes (default) or creating secondaries for the non-padding fields.
+  ConfigFile: &ConfigFile {^Parameter: {Name: "ConfigurationFile", Default: "../../phases/execution/config/MultiDeletes/Default.yml"}}
+
+  ##### Constants ####
+
+  WriteRate: &WriteRate 1 per 500 microseconds  # 2000/second
+  ConcurrentWritesColl: &ConcurrentWritesColl concurrentWritesColl
+  Database: &Database test
+  MassDeletionsColl: &MassDeletionsColl Collection0
+  NumPhases: &NumPhases 11
+  LoaderThreads: &LoaderThreads 4
+  # Delete by primary index always.
+  DeleteQuery: &DeleteQuery
+    {_id: {$gte: {^ObjectId: "000000000000000000000000"}}}
+  # Only used for generating the Document padding.
+  RandomStringLength: &RandomStringLength 6
+
+  #### Document Size Specifications ####
+
+  # __________  1KB Specifications __________
+  TargetDocumentSizeBytes_1KB: &TargetDocumentSizeBytes_1KB 1_024  # 1KB
+  DocumentCount_1KB: &DocumentCount_1KB 10_000_000
+  InsertBatchSize_1KB: &InsertBatchSize_1KB 1_000
+  DocumentPaddingSizeBytes_1KB: &DocumentPaddingSizeBytes_1KB
+    ^NumExpr:
+      withExpression: "target_size - id_length - (field_count * (field_length+value_length)) - padding_field"
+      andValues: {target_size: *TargetDocumentSizeBytes_1KB, id_length: 22, field_count: 7, field_length: 8, value_length: *RandomStringLength, padding_field: 14}
+  # Insert same document for both default and with secondary indexes to remove one variable.
+  DocumentToInsert_1KB: &DocumentToInsert_1KB
+    a: &rs_1KB {^FastRandomString: {length: *RandomStringLength, alphabet: "0123456789"}}
+    b: *rs_1KB
+    c: *rs_1KB
+    d: *rs_1KB
+    e: *rs_1KB
+    f: *rs_1KB
+    g: *rs_1KB  # up to field_count fields
+    padding: {^FastRandomString: {length: *DocumentPaddingSizeBytes_1KB}}
+
+  # __________  10MB Specifcations __________
+  TargetDocumentSizeBytes_10MB: &TargetDocumentSizeBytes_10MB 10_485_760  # 10MB
+  DocumentCount_10MB: &DocumentCount_10MB 4_000
+  InsertBatchSize_10MB: &InsertBatchSize_10MB 1
+  DocumentPaddingSizeBytes_10MB: &DocumentPaddingSizeBytes_10MB
+    ^NumExpr:
+      withExpression: "target_size - id_length - (field_count * (field_length+value_length)) - padding_field"
+      andValues: {target_size: *TargetDocumentSizeBytes_10MB, id_length: 22, field_count: 7, field_length: 8, value_length: *RandomStringLength, padding_field: 14}
+  # Insert same document for both default and with secondary indexes to remove one variable.
+  DocumentToInsert_10MB: &DocumentToInsert_10MB
+    a: &rs_10MB {^FastRandomString: {length: *RandomStringLength, alphabet: "0123456789"}}
```

Comment:
`rs_1KB` is the same as `rs_10MB`, would it make sense to use the same definition?